generators/server/templates/src/main/java/package/security/oauth2/JwtAuthorityExtractor.java.ejs

```diff
@@ -34,6 +34,9 @@ public class JwtAuthorityExtractor extends JwtAuthenticationConverter {
 
     @Override
     protected Collection<GrantedAuthority> extractAuthorities(Jwt jwt) {
-        return SecurityUtils.extractAuthorityFromClaims(jwt.getClaims());
+        Collection<GrantedAuthority> grantedAuthorities = super.extractAuthorities(jwt);
```

Comment:
To be consistent, similar code should also be added in `GrantedAuthoritiesMapper`